pkg/apis/apps/v1alpha1/workloadrebalancer_types.go

```diff
@@ -57,6 +57,14 @@ type WorkloadRebalancerSpec struct {
 	// +kubebuilder:validation:MinItems=1
 	// +required
 	Workloads []ObjectReference `json:"workloads"`
+
+	// ttlMinutesAfterFinished limits the lifetime of a WorkloadRebalancer that has finished execution (means each
+	// target workload is finished with result of Successful or Failed).
+	// If this field is set, ttlSecondsAfterFinished after the WorkloadRebalancer finishes, it is eligible to be automatically deleted.
+	// If this field is unset, the WorkloadRebalancer won't be automatically deleted.
+	// If this field is set to zero, the WorkloadRebalancer becomes eligible to be deleted immediately after it finishes.
+	// +optional
+	TTLMinutesAfterFinished *int32 `json:"ttlMinutesAfterFinished,omitempty"`
```

Comment:
Just a question, why not use `seconds` but `minutes` here?

Reply:
In my expected usage scenarios of `WorkloadRebalancer`, users do not need to be accurate to seconds, only to minutes.

I think `rebalance` is a key operation. In actual usge, it may take more than ten minutes to observe whether the status of the instance meets expectations after the operation. If there is an exception, it may take another ten minutes to use this resource object to locate the problem. So the deletion time must be in minutes.

Reply:
besides, since the usage scenario is `minites` level, `minutes` may be more readable than `seconds`.